firmware/Core/Src/telecommands/i2c_telecommand_defs.c

```diff
@@ -0,0 +1,52 @@
+#include "i2c_telecommand_defs.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+uint8_t TCMDEXEC_scan_i2c_device(const char *args_str, TCMD_TelecommandChannel_enum_t tcmd_channel,
+                        char *response_output_buf, uint16_t response_output_buf_len) {
+
+    snprintf(response_output_buf, response_output_buf_len, "Starting I2C scan...\n");
+
+    I2C_HandleTypeDef* hi2c;
+    int bus_to_scan = atoi(args_str);
```

Comment:
What happens if this fails? Also, please use `uint8_t` (or similar specific type) instead of `int`

Reply:
If the conversion fails atoi returns 0 which is handled by the default case in the switch statement. Should this be handled explicitly before we hit the switch statement?